### PR TITLE
support zoom

### PR DESCRIPTION
**Pull request by iloveitaly**
### All submissions

* [x] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
* [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change

### Adding/updating Application X Support

* [x] This PR is only for one application
* [x] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
* [x] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
* [x] Syncing does not break the application
* [x] Syncing does not compete with any syncing functionality internal to the application
* [x] The configuration syncs the minimal set of data
* [x] No file specific to the local workstation is synced
* [x] No sensitive data is synced

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -712,6 +712,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [zathura](https://pwmt.org/projects/zathura/)
 - [zoxide](https://github.com/ajeetdsouza/zoxide)
 - [Zsh](http://zsh.sourceforge.net/)
+- [Zoom](http://zoom.com/)
```
Comment (lra) [inline README.md:715]: Can you keep this list AZ ordered?

**Comment by iloveitaly** [comment]
@Ira Fixed sorting!

**Comment by iloveitaly** [comment]
Friendly bump here @IrA!

**Review by lra — APPROVED** [review]
Thanks!

**Pull request merged**